### PR TITLE
added anchor for table of contents

### PR DESCRIPTION
**Pull request by tobiaskresse**

### CODE REVIEW

File: Resources/Private/Templates/Bibliografie/List.html [hunk 1]
```diff
@@ -15,9 +15,23 @@
...
+          <f:for each="{collections}" as="collection" iteration="collectionIterator">
+            <f:if condition="{collection.data.title}">
+            <li><f:link.page section="{collection.data.title}{collectionIterator.cycle}">{collection.data.title}</f:link.page></li>
+            </f:if>
+          </f:for>
+        </ul>
+        </f:then>
+    </f:if>
+    <f:else>
+    <f:debug>{settings}</f:debug>
```
Comment (albig) [inline Resources/Private/Templates/Bibliografie/List.html]:
```suggestion
```

**Review by albig — CHANGES_REQUESTED** [review]
Please remove debug output

File: Resources/Private/Templates/Bibliografie/List.html [hunk 2]
```diff
@@ -15,9 +15,23 @@
 <f:section name="main">
   <f:flashMessages />
   <div class="tx-slub-zotero-collections">
-      <f:for each="{collections}" as="collection">
+    <f:if condition="{settings.zotero.tableOfContents} == 1">
+      <f:debug>{_all}</f:debug>
```
Comment (ByteParty) [inline Resources/Private/Templates/Bibliografie/List.html]: please remove the debug code.

File: Resources/Private/Templates/Bibliografie/List.html [hunk 3]
```diff
@@ -15,9 +15,23 @@
...
+          </f:for>
+        </ul>
+        </f:then>
+    </f:if>
+    <f:else>
+    <f:debug>{settings}</f:debug>
+    <f:for each="{collections}" as="collection" iteration="collectionIterator">
         <f:if condition="{collection.data.title}">
-          <h3 class="tx-slub-zotero-collection-title">{collection.data.title}</h3>
+          <h3 class="tx-slub-zotero-collection-title" id="c{collection.data.title}{collectionIterator.cycle}">{collection.data.title}</h3>
```
Comment (ByteParty) [inline Resources/Private/Templates/Bibliografie/List.html]: Please do not use a title as id. Use only the iterator or a zotero id.

**Review by ByteParty — CHANGES_REQUESTED** [review]

**Review by albig — APPROVED** [review]

**Pull request merged**